### PR TITLE
added root_help for --help options on scripts

### PR DESCRIPTION
**Pull request by mjmottram**
--help on scripts previously crashed (ROOT import issue).

Also changed the definition of echidna_home - previously this was whatever the user's working directory was, assumed that we actually want it to be the base dir of echidna, is this correct?

### CODE REVIEW

File: echidna/core/root_help.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+''' Helper script to allow --help option to work with ROOT import.
```
Comment (ashleyrback) [inline echidna/core/root_help.py]: Can you write it as ``... allow :option:`--help` option...`` please, so this shows up nicer in the build documentation

File: echidna/core/fill_spectrum.py [hunk 2]
```diff
@@ -1,3 +1,4 @@
+import root_help
```
Comment (ashleyrback) [inline echidna/core/fill_spectrum.py]: I'm slightly confused why you've imported it here, as this script does not take command line arguments, so no `--help` option anyway.

File: echidna/output/plot_root.py [hunk 3]
```diff
@@ -1,3 +1,4 @@
+from echidna.core import root_help
```
Comment (ashleyrback) [inline echidna/output/plot_root.py]: Same as above, there are also no command line args here. Am I missing something in the way this works?

File: echidna/core/fill_spectrum.py (hunk 2 above)
Comment (mjmottram) [inline echidna/core/fill_spectrum.py]: Needs to be imported by any module that also imports ROOT (I think, was a while ago so I can't recall the reasoning).

Comment (ashleyrback) [inline echidna/core/fill_spectrum.py]: Ah OK, so it is imported where rat or ROOT is imported, before their imports, and not in the module that parses the arguments.

Comment (mjmottram) [inline echidna/core/fill_spectrum.py]: Yup.

Comment (ashleyrback) [inline echidna/core/fill_spectrum.py]: Also I would prefer not to have relative imports as they make it less clear what you are importing and where it's coming from see [more detailed arguments]( http://programmers.stackexchange.com/questions/159503/whats-wrong-with-relative-imports-in-python). But it's up to you if you want to change it or not.

**Comment by ashleyrback** [comment]
Finally what was the reasoning behind putting the `root_help` module in `core/`? I'm not certain this is the best place for it, but I'm also not sure where a better place would be? It feels like something you'd have in a kind of utility module. Any thoughts?

**Comment by mjmottram** [comment]
Sure, makes sense to have a utilities folder.

**Comment by ashleyrback** [comment]
Happy, merging!

**Pull request merged**